test/js/dropdown-utils_test.js

```diff
@@ -38,6 +38,40 @@ describe('Dropdown utils', function() {
     expect($('option')).to.have.length(4);
   });
 
+  it('should not add options with same values', function() {
```

Comment:
There's one option already, before I call addOption twice. It should be in before or beforeEach